test/unit/spec/localparticipant.js

```diff
@@ -151,6 +155,123 @@ describe('LocalParticipant', () => {
     });
   });
 
+  describe('#setBandwidthProfile', () => {
+    const subscriptionModes = Object.values(subscriptionMode);
+    const trackPriorities = [PRIORITY_HIGH, PRIORITY_LOW, PRIORITY_STANDARD];
+
+    [
+      [null, 'that is null', TypeError, 'object'],
+      ['foo', 'that is not an object', TypeError, 'object'],
+      [['bar'], 'that is an Array', TypeError, 'object'],
+      [{ video: null }, 'whose .video is null', TypeError, 'object'],
+      [{ video: 'baz' }, 'whose .video is not an object', TypeError, 'object'],
+      [{ video: ['zee'] }, 'whose .video is an Array', TypeError, 'object'],
+      [{ video: { dominantSpeakerPriority: 2 } }, `whose .video.dominantSpeakerPriority is not one of ${trackPriorities.join(', ')}`, RangeError, trackPriorities],
+      [{ video: { maxSubscriptionBitrate: false } }, 'whose .video.maxSubscriptionBitrate is not a number', TypeError, 'number'],
+      [{ video: { maxTracks: {} } }, 'whose .video.maxTracks is not a number', TypeError, 'number'],
+      [{ video: { mode: 'foo' } }, `whose .video.mode is not one of ${subscriptionModes.join(', ')}`, RangeError, subscriptionModes],
+      [{ video: { renderDimensions: null } }, 'whose .video.renderDimensions is null', TypeError, 'object'],
+      [{ video: { renderDimensions: true } }, 'whose .video.renderDimensions is not an object', TypeError, 'object'],
```

Comment:
please add `{ video: { renderDimentions: undefined } }` as a test case

Reply:
@makarandp0 I've already added unit tests for the cases you mentioned in the last commit.